### DISCUSSION TITLE
Issue with Aux adapter after power outage

### DISCUSSION THREAD

**Discussion by mzuniga51** [tagyoureit/nodejs-poolController#459] [General]
@rstrouse Hi.   Whenever there are power outages (frequent here where I live) I have to usually restart the two RS485 adapters as well as the Intellicenter and my dedicated Mac.  Nothing too strange there but in a couple of occasions after that I loose the aux chlorinator completely, meaning I have to add it back in njsPC.  It only happens after power outages and never on reset.  Any ideas? Thanks.

**Comment by rstrouse** [comment]
If it happens again upload a replay.  Unless you are corrupting the poolConfig.json I can’t see a scenario where this would happen.
